### PR TITLE
Move the promptflow models into assets/promptflow/models

### PR DESCRIPTION
**Pull request by dorisjoy**
Move the promptflow models into assets/promptflow/models with below refines:
1. Refine the model properties;
2. Refine the model description;
3. Add two new models;

### CODE REVIEW

**Review by lykelly19 — APPROVED** [review]

**Comment by skasturi** [comment]
Is it expected for `model.yaml` to be empty?

File: assets/promptflow/models/classification_accuracy_eval/description.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+### What you will learn
+
+In this flow, you will learn
+- how to compose a point based evaluation flow, where you can calculate point-wise metrics.
+- the way to log metrics.
+
+#### Evaluate a classification flow
+There are two ways to evaluate an classification flow.
+* Run a classification flow and evaluation flow all together
+    * step 1: create or clone an classification flow
```
Comment (novaturient95) [inline assets/promptflow/models/classification_accuracy_eval/description.md:14]:
```suggestion
    * step 1: create or clone a classification flow
```

File: assets/promptflow/models/classification_accuracy_eval/description.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+This is a flow illustrating how to evaluate the performance of a classification system. It involves comparing each prediction to the groundtruth and assigns a "Correct" or "Incorrect" grade, and aggregating the results to produce metrics such as accuracy, which reflects how good the system is at classifying the data.
```
Comment (novaturient95) [inline assets/promptflow/models/classification_accuracy_eval/description.md:1]: Wondering why description talks about eval and does not give any information about model to be published?
Could we add description for model first followed by steps to trigger evaluation?

File: assets/promptflow/models/bring_your_own_data_qna/description.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+This is a flow demonstrating Q&A with GPT3.5 using data from your own indexed files to make the answer more grounded. It involves embedding the raw query, using "Vector Search" to find most relevant context in user data, and then using GPT3.5 to generate an answer to the question with the documents. This example also contains multiple prompt variants that you can tune.
```
[the remark this answers is not in the document]
Comment (dorisjoy) [reply assets/promptflow/models/bring_your_own_data_qna/description.md]: We are going to add total more than twenty promptflow models later. And we also need to add another asset type for promptflow tools (WIP). We want to put these all under promptflow folder for better management.

File: assets/promptflow/models/classification_accuracy_eval/description.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Brief description: Measuring the performance of a classification system by comparing its outputs to groundtruth.
+
+### What you will learn
+
+In this flow, you will learn
+- how to compose a point based evaluation flow, where you can calculate point-wise metrics.
+- the way to log metrics.
+
+#### Evaluate a classification flow
+There are two ways to evaluate an classification flow.
```
[the remark this answers is not in the document]
Comment (dorisjoy) [reply assets/promptflow/models/classification_accuracy_eval/description.md]: Fixed

File: assets/promptflow/models/classification_accuracy_eval/description.md (hunk 1 above)
Comment (dorisjoy) [reply assets/promptflow/models/classification_accuracy_eval/description.md]: fixed

[the remark this answers is not in the document]
Comment (dorisjoy) [reply assets/promptflow/models/classification_accuracy_eval/description.md]: The promptflow model will be shown as examples under promptflow tab, and there user can click 'view detail' to create/clone a flow. Currently the description is just a draft one, PM is working on the official version, we will update it later.
image

File: assets/promptflow/models/classification_accuracy_eval/description.md (hunk 2 above)
Comment (dorisjoy) [reply assets/promptflow/models/classification_accuracy_eval/description.md]: Currently the description is just a draft one, PM is working on the official version, we will update it later.

**Review by novaturient95 — APPROVED** [review]

**Review by lykelly19 — APPROVED** [review]

**Pull request merged**